packages/health-check/src/verboseChecks.js

```diff
@@ -0,0 +1,283 @@
+    skylink: "AADW6GsQcetwDBaDYnGCSTbYjSKY743NtY1A5VRx5sj3Dg/",
+    metadataHash: "b17a16a3522f2877b5b9ebdcdfd98956eade8721",
+    bodyHash: "077e54054748d278114f1870f8045a162eb73641",
+  };
+
+  skylinkVerification(done, linkInfo);
+}
+
+// skyGalleryIndexFileCheck returns the result of trying to download the skylink
+// for the SkyGallery Application with a trailing /index.html
+function skyGalleryIndexFileCheck(done) {
+  const linkInfo = {
+    description: "SkyGallery Index File",
+    skylink: "AADW6GsQcetwDBaDYnGCSTbYjSKY743NtY1A5VRx5sj3Dg/index.html",
+    metadataHash: "203dcc63e6c3db7501822d9f0d38fab26f3dbceb",
+    bodyHash: "077e54054748d278114f1870f8045a162eb73641",
+  };
+
+  skylinkVerification(done, linkInfo);
+}
+
+// skyGalleryRedirectCheck returns the result of trying to download the skylink
+// for the SkyGallery Application with no trailing slash.
+function skyGalleryRedirectCheck(done) {
+  const linkInfo = {
+    description: "SkyGallery Redirect",
+    skylink: "AADW6GsQcetwDBaDYnGCSTbYjSKY743NtY1A5VRx5sj3Dg",
+    metadataHash: "b17a16a3522f2877b5b9ebdcdfd98956eade8721",
+    bodyHash: "077e54054748d278114f1870f8045a162eb73641",
+  };
+
+  skylinkVerification(done, linkInfo);
+}
+
+// uniswapIndexFileCheck returns the result of trying to download the skylink
+// for the Uniswap Application with a trailing /index.html
+function uniswapIndexFileCheck(done) {
+  const linkInfo = {
+    description: "Uniswap Skylink Index File",
+    skylink: "IAC6CkhNYuWZqMVr1gob1B6tPg4MrBGRzTaDvAIAeu9A9w/index.html",
+    metadataHash: "5baef23f7cc138a41d945727eb862fc2cd94ae45",
+    bodyHash: "3965f9a7def085b3a764ddc76a528eda38d72359",
+  };
+
+  skylinkVerification(done, linkInfo);
+}
+
+// skylinkVerification verifies a skylink against known information provided in
+// the linkInfo.
+function skylinkVerification(done, linkInfo) {
+  const time = process.hrtime();
+
+  // Create the query for the skylink
+  let query = `http://${process.env.PORTAL_URL}/${linkInfo.skylink}?nocache=true`;
+
+  // Get the Skylink
+  superagent
+    .get(query)
+    .responseType("blob")
+    .end((err, res) => {
+      // Record the statusCode
+      const statusCode = (res && res.statusCode) || (err && err.statusCode) || null;
+
+      // Determine if the skylink is up. Start with checking if there was an
+      // error in the request.
+      let up = err === null;
+      if (up) {
+        // Check if the response body is valid by checking against the known
+        // hash
+        let validBody = hash(res.body) === linkInfo.bodyHash;
+        // Check if the metadata is valid by checking against the known
+        // hash
+        let validMetadata = hash(res.header["skynet-file-metadata"]) === linkInfo.metadataHash;
+        // Redetermine if the Skylink is up based on the results from the body
+        // and metadata hash checks
+        up = up && validBody && validMetadata;
+      }
+
+      // Return the entry information
+      done({
```

Comment:
@kwypchlo currently none of these are critical checks. How do we get the results of them so that we can verify the results when starting a server and testing code such as on the dev server?

Reply:
those will be printed to `/health-check`

Reply:
I'll add it to the issue :+1: